WordPress/Classes/ViewRelated/Stats/Subscribers/StatsSubscribersViewModel.swift

```diff
@@ -88,6 +91,40 @@ private extension StatsSubscribersViewModel {
     }
 }
 
+// MARK: - Subscribers List
+
+private extension StatsSubscribersViewModel {
+    func subscribersListRows() -> [any StatsHashableImmuTableRow] {
+        switch store.subscribersList.value {
+        case .loading, .idle:
+            return loadingRows(.subscribersList)
+        case .success(let subscribers):
+            return [
+                TopTotalsPeriodStatsRow(
+                    itemSubtitle: StatSection.ItemSubtitles.subscriber,
+                    dataSubtitle: StatSection.DataSubtitles.since,
+                    dataRows: subscribersListDataRows(subscribers),
+                    statSection: .subscribersList,
+                    siteStatsPeriodDelegate: viewMoreDelegate
+                )
+            ]
+        case .error:
+            return errorRows(.subscribersList)
+        }
+    }
+
+    func subscribersListDataRows(_ subscribers: [StatsFollower]) -> [StatsTotalRowData] {
+        return subscribers.map {
+            return StatsTotalRowData(
+                name: $0.name,
+                data: $0.subscribedDate.relativeStringInPast(),
```

Comment:
> The web does a good job IMO for other cases like "3 minutes" (the app shows "< 1 hour" though. For a high-traffic site, I can see new followers on every pull-to-refresh, so seeing the time since in minutes is nice. This is out of scope, so I'm just leaving it as an observation.

Good observation. Maybe we could use `RelativeDateTimeFormatter` instead of our custom implementation. It produces a localized relative string. It's used in the Latest Post Summary card:

<img width="359" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/737a775f-bf2f-453b-a18c-2a3e7f1d6f7f">

However, in the Latest Post Summary card case it produces a bit longer string than we need, add it as a draft task to try out if there'll be capacity.

Reply:
Good thought about `RelativeDateTimeFormatter` 👍